cmd/plume/specs.go

```diff
@@ -207,6 +207,56 @@ var (
 				},
 			},
 		},
+		"developer": channelSpec{
+			BaseURL: "gs://builds.developer.core-os.net/boards",
+			Boards:  []string{"amd64-usr", "arm64-usr"},
+			Destinations: []storageSpec{storageSpec{
+				BaseURL:       "gs://builds.developer.core-os.net/releases",
+				NamedPath:     "current",
+				VersionPath:   true,
+				DirectoryHTML: true,
+				IndexHTML:     true,
+			}},
+			GCE: gceSpec{
+				Project:     "coreos-gce-testing",
+				Family:      "coreos-dev",
```

Comment:
Should be `coreos-developer`, and probably the other one should be changed to `coreos-user`.